src/tests/baseservices/exceptions/UnhandledExceptionHandler/Foreign/ForeignUnhandled.cs

```diff
@@ -9,9 +9,9 @@
 
 public delegate void MyCallback();
 
-public class PInvokeRevPInvokeUnhandled
+public class ForeignUnhandled
 {
-    [DllImport("ForeignThreadRevPInvokeUnhandled")]
```

Comment:
Not sure how this dllimport resolved before when the file and cmake used `ForeignThreadRevPInvokeUnhandledNative`